scripts/hudson/quickstart.sh

```diff
@@ -140,15 +140,19 @@ function configure_wildfly {
   cd $WORKSPACE
   wget --user=guest --password=guest -nv https://ci.wildfly.org/httpAuth/repository/downloadAll/WF_Nightly/.lastSuccessful/artifacts.zip
   unzip -q artifacts.zip
-  export WILDFLY_DIST_ZIP=$(ls wildfly-*Final-SNAPSHOT.zip | head -n 1)
-  [ "x$WILDFLY_DIST_ZIP" = "x" ] && echo "Cannot find WildFly distribution zip file" && return 1
-  unzip -q $WILDFLY_DIST_ZIP
-  export JBOSS_HOME=`pwd`/${WILDFLY_DIST_ZIP%.zip}
+  # the artifacts.zip may be wrapping several zip files: artifacts.zip -> wildfly-latest-SNAPSHOT.zip -> wildfly-###-SNAPSHOT.zip
+  local wildflyLatestZipWrapper=$(ls wildfly-latest-*.zip | head -n 1)
+  unzip -q $wildflyLatestZipWrapper
+  rm -f $wildflyLatestZipWrapper
+  local wildflyDistZip=$(ls wildfly-*-SNAPSHOT.zip | head -n 1)
+  [ "x$wildflyDistZip" = "x" ] && echo "Cannot find WildFly distribution zip file" && return 1
+  unzip -q ${wildflyDistZip}
+  export JBOSS_HOME=`pwd`/${wildflyDistZip%.zip}
   cp $JBOSS_HOME/docs/examples/configs/standalone-xts.xml $JBOSS_HOME/standalone/configuration/
   cp $JBOSS_HOME/docs/examples/configs/standalone-rts.xml $JBOSS_HOME/standalone/configuration/
   # cleaning
   rm -f artifacts.zip
-  rm -rf "${WILDFLY_DIST_ZIP}"
+  rm -rf "${wildflyDistZip}"
```

Comment:
Did you mean `rm -rf ${JBOSS_HOME}?
Also what about files matching the pattern `wildfly-*-SNAPSHOT.zip`

Reply:
no, I meant removing the zip file to "release" the space at the drive. The `JBOSS_HOME` is needed for further execution of the tests. Using the matching pattern makes sense. I changed that.